tests/cluster/tests/12-replication.tcl

```diff
@@ -0,0 +1,61 @@
+# Replication test.
+#
+# Check CLUSTER REPLICATE commands.
+
+source "../tests/includes/init-tests.tcl"
+
+proc find_slots_for_node {node_idx node_id_to_find} {
```

Comment:
This is the old style test, the new style is in unit/cluster. These tests have a lot of missing debugging information and also aren't run as part of the CI since they don't get parallelized. 

The tests should port mostly cleanly, the main difference is that you need to call `create_cluster 3 6` at the start but most of the other helps still exist.

Reply:
moved to ````tests/unit/cluster/replica-detach.tcl````